Added landmarks example and added links for the landmarks section of the APG

jongund opened the pull request:
I have updated the landmarks examples based on the May 16th APG teleconference and the changes are ready for review by the group

MichielBijl commented:
Thanks Jon, I'll review this week. Shall we discuss it during the call on the 16th?

jongund commented:
Sure.  The 16th is good for me.

From: Michiel Bijl <notifications@github.com<mailto:notifications@github.com>>
Reply-To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, May 8, 2016 at 2:30 PM
To: w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)


Thanks Jon, I'll review this week. Shall we discuss it during the call on the 16th?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_39-23issuecomment-2D217741162&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=yekSgfETPU2_pwm4VaymG5TYUH-ymxoTnshg_HjiQP8&s=O1w4aH2qFFOd8Rl2JlaRlopGVMFYvrdkmfMx5lskFIo&e=>

jongund commented:
Just updated assistive technology support page for ARIA landmarks

MichielBijl commented:
Thanks for the update Jon!

MichielBijl commented:
Comments after review:

- We decided that text in landmark code examples should not contain APG text. Can you replace this with something else?
- Main navigation on the left hand side of the page uses a tablets role, but does not implement APG tabs design pattern.
- Please audit the rest of the examples yourself first.

jongund commented:
Michiel,

Here is the latest draft of the landmark example that implements most of the comments from the May 16th call:
https://rawgit.com/jongund/oaa-examples/master/examples/landmarks/index.html

I just need to finish the form example page.
Comments on the changes are welcome.

Response to your comments:

  *   We decided that text in landmark code examples should not contain APG text. Can you replace this with something else?
     *   The group felt is was OK in this case of landmarks, and I am committed to making sure the example stays synced with the APG, since I use the example in training
  *   Main navigation on the left hand side of the page uses a tablets role, but does not implement APG tabs design pattern.
     *   I have fixed this problem, it is a problem with Bootstrap.js that needs to be repaired
  *   Please audit the rest of the examples yourself first.
     *   I am using AInspector Sidebar for Firefox to test the examples and fixing the bugs I find: https://addons.mozilla.org/en-US/firefox/addon/ainspector-sidebar/

Jon


From: Michiel Bijl <notifications@github.com<mailto:notifications@github.com>>
Reply-To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, May 23, 2016 at 6:42 AM
To: w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)


Comments after review:

  *   We decided that text in landmark code examples should not contain APG text. Can you replace this with something else?
  *   Main navigation on the left hand side of the page uses a tablets role, but does not implement APG tabs design pattern.
  *   Please audit the rest of the examples yourself first.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_39-23issuecomment-2D220957612&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=7VojjhmTipvKY3719jJNqzSW1kRwhgUMwAgakLoLm3s&s=t6vbZUnS3IHOoKBETTPXtoiqNJMO45xHxwDsnyMo5-A&e=>

MichielBijl commented:
Okay Jon, thanks for the reply! If you feel the examples are good give me a sign and I'll merge this in.

jongund commented:
Need to finish the form landmark example.

Jon


From: Michiel Bijl <notifications@github.com<mailto:notifications@github.com>>
Reply-To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, May 24, 2016 at 8:01 PM
To: w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)


Okay Jon, thanks for the reply! If you feel the examples are good give me a sign and I'll merge this in.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_39-23issuecomment-2D221445890&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=4eVyXMNnD09pWX0qPmmTwdkot4eQVG1HK3bSi4oHQUo&s=014MLawX0yMMaAxieaoN7lFSz8nDYvRlSATV72H6064&e=>

jongund commented:
Michiel,

I have added the form landmark example, plus made changes based on the May 16th teleconference, so I think it now ready to merge and give people another chance for comment.

Jon


From: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>
Date: Thursday, May 26, 2016 at 7:55 AM
To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>, w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)

Need to finish the form landmark example.

Jon


From: Michiel Bijl <notifications@github.com<mailto:notifications@github.com>>
Reply-To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, May 24, 2016 at 8:01 PM
To: w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)


Okay Jon, thanks for the reply! If you feel the examples are good give me a sign and I'll merge this in.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_39-23issuecomment-2D221445890&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=4eVyXMNnD09pWX0qPmmTwdkot4eQVG1HK3bSi4oHQUo&s=014MLawX0yMMaAxieaoN7lFSz8nDYvRlSATV72H6064&e=>

MichielBijl commented:
Thanks Jon!

jongund commented:
Michiel,

Thank you for the merge, but I found a few bugs that need to be fixed this morning, so I created another merge request.

Jon


From: Michiel Bijl <notifications@github.com<mailto:notifications@github.com>>
Reply-To: w3c/aria-practices <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, May 26, 2016 at 7:46 PM
To: w3c/aria-practices <aria-practices@noreply.github.com<mailto:aria-practices@noreply.github.com>>
Cc: Jon Gunderson <jongund@illinois.edu<mailto:jongund@illinois.edu>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [w3c/aria-practices] Added landmarks example and added links for the landmarks section of the APG (#39)


Thanks Jon!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_w3c_aria-2Dpractices_pull_39-23issuecomment-2D222033483&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=u3yC5IynZEw6n4DopNuUEX2A6FrSASyCpmin8keFiPE&s=8zz79ZdCpP0wv1CXxDlHg5ibEbc2O4DIMiaUjE8yRqY&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_ABcE7BLx2T0sEdrkTuKZDy64pnbh-5FPivks5qFj7lgaJpZM4IZWRj&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=REZD8fc2AwufInstfW3L5jSLVS8bjZtAodDOhat7yAI&m=u3yC5IynZEw6n4DopNuUEX2A6FrSASyCpmin8keFiPE&s=hp5jopv6__JIYTutW2yEkYAIyD2F3b1CuQ-FCjS9cXI&e=>.
